src/components/Analytics/utils.ts

```diff
@@ -34,6 +34,7 @@ export const legacyTracker: Tracker = {
 export enum EventCategory {
   API = 'api',
   ARTICLES = 'articles',
+  BANNER = 'banner',
```

Comment:
Is this a good idea?  I wasn't sure what category to use.  It seems in the past we've created new categories for banners (e.g. VOTE_2020).  But I wasn't sure.

Reply:
changed to HOMEPAGE_BANNER